supplier_management_site/settings/base.py

```diff
@@ -174,7 +174,7 @@
 CELERY_TASK_SERIALIZER = 'json'
 CELERY_BROKER_URL = BROKER_URL
 CELERY_RESULT_BACKEND = BROKER_URL
-CELERY_ALWAYS_EAGER = False
+CELERY_ALWAYS_EAGER = True
```

Comment:
Is it ok to be True on base? how is it going to affect production?
https://docs.celeryproject.org/en/3.1/configuration.html#task-execution-settings